src/meta-srv/src/service/heartbeat.rs

```diff
@@ -67,11 +67,11 @@ impl heartbeat_server::Heartbeat for Metasrv {
                             break;
                         };
 
-                        if pusher_key.is_none() {
-                            pusher_key = register_pusher(&handler_group, header, tx.clone()).await;
+                        if pusher_id.is_none() {
+                            pusher_id = register_pusher(&handler_group, header, tx.clone()).await;
                         }
-                        if let Some(k) = &pusher_key {
-                            METRIC_META_HEARTBEAT_RECV.with_label_values(&[k]);
+                        if let Some(k) = &pusher_id {
+                            METRIC_META_HEARTBEAT_RECV.with_label_values(&[&format!("{k:?}")]);
```

Comment:
This use the debug format. Is it expected? Maybe we should use display.
```suggestion
                            METRIC_META_HEARTBEAT_RECV.with_label_values(&[&k.to_string()]);
```